src/testing/cluster/storage_checker.zig

```diff
@@ -171,5 +177,42 @@ pub fn StorageCheckerType(comptime Storage: type) type {
             }
             return checksum;
         }
+
+        fn checksum_grid(checker: *Self, superblock: *const SuperBlock) u128 {
```

Comment:
At long last, the storage checker can test grid storage determinism at checkpoint! (When sync isn't running, that is.)

Reply:
:tada: